client/tracing/src/logging/mod.rs

```diff
@@ -220,6 +222,15 @@ impl LoggerBuilder {
 		self
 	}
 
+	/// Add a custom profiler.
+	pub fn with_custom_profiling(
+		&mut self,
+		custom_profiler: Box<dyn crate::TraceHandler>,
```

Comment:
I mean you could have left the vector :P It was just about not taking a vec directly. But fine.

Reply:
IMO it doesn't make sense to have many of these, one is enough, since you can still multiplex further. However, we can change it to vec later if there is demand 😁